src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/instructions/instructions.rs

```diff
@@ -89,6 +89,17 @@ impl Instructions {
         )
         .map_err(|e| extract_error_from_error(e, network_id))
         .and_then(|manifest| {
+            // This catches and fixes "broken invariants"
+            std::panic::catch_unwind(|| {
```

Comment:
@iamyulong @0xOmarA   is this the correct thing to do? Or is this example manifest "valid" but `summary` is broken? hmm I would appreciate any input. We are about to do a release soon, so maybe we wont have time to bump RET / Scrypto dependencies before our next release. cc @GhenadieVP

Reply:
reverted addition of `catch_unwind` since newer version of RET does not crash,